packages/components/src/toolbar/toolbar-group/style.scss

```diff
@@ -14,6 +13,10 @@
 		margin: 0;
 	}
 
+	.components-accessible-toolbar:not(.zoom-out-toolbar) & {
+		background: $white;
```

Comment:
I think it would be better to just remove this altogether, but there is a risk that it's used for something...